### ISSUE TITLE
Can tf.keras.utils.normalize be verified by tf.norm?

### ISSUE THREAD

**Issue by kathy-lee** [tensorflow/tensorflow#40460]
I tried using `tf.keras.utils.normalize` to normalize the MNIST image dataset and then checking the normalized result with `tf.norm` on an arbitrary image (e.g. the 11th in the following sample) with:

```
(trainX, trainY), (testX, testY) = mnist.load_data()
print(trainX.shape)
print(np.amax(trainX[11]))  
trainX = tf.keras.utils.normalize(trainX,axis=1)
print(np.amax(trainX[11]))
print(tf.norm(trainX[11]))
```

but it gave me the following result:

```
(60000, 28, 28)
255
0.88178858555868
tf.Tensor(4.47213595499958, shape=(), dtype=float64)
```

I thought after normalization the max value of an arbitrary image should be `1` which is actually `0.88178858555868`. On the other hand, the norm of the normalized image should be `1` while it is `4.47213595499958`. Is my understanding wrong? Can anyone help me please?

**Comment by jvishnuvardhan** [comment]
@kathy-lee This is not the normalization you are expecting. Please check this [stackoverflow answer](https://stackoverflow.com/questions/52571752/what-is-the-purpose-of-keras-utils-normalize) for more information. 

Please post this kind of support questions (that are not related to bug/performance/feature) in Stackoverflow as there is a large community to support and depending on requirement we will also answer questions there. Thanks!

**Comment by kathy-lee** [comment]
@jvishnuvardhan Thanks for your reply. If it is L2 normalization, then after keras.utils.normalize() and tf.norm() we should get 1.
But apparently it is not. as I wrote above, I got `tf.Tensor(4.47213595499958, shape=(), dtype=float64)`. So I think it's either a bug in the normalization calculation (because it could not be verified by `tf.norm()`), or an error in its [document](https://www.tensorflow.org/api_docs/python/tf/keras/utils/normalize) because it's not really a L2 norm when `order=2`.

**Comment by jvishnuvardhan** [comment]
Please note that `keras.utils.normalize` and `tf.norm` are different. Please post support related questions in Stackoverflow. GitHub is mainly for bugs and performance related issues. 

I am closing this issue. Please feel free to post it in StackOverflow. Thanks!
